homeassistant/components/simplisafe/alarm_control_panel.py

```diff
@@ -13,7 +13,7 @@
     EVENT_ARMED_HOME,
     EVENT_AWAY_EXIT_DELAY_BY_KEYPAD,
     EVENT_AWAY_EXIT_DELAY_BY_REMOTE,
-    EVENT_DISARMED_BY_MASTER_PIN,
+    EVENT_DISARMED_BY_KEYPAD,
```

Comment:
This change doesn't relate to this PR. I'd recommend a separate PR that (a) bumps `simplisafe-python` and (b) makes this change. Then, rebase this PR on top of that one (once it's merged into `dev`).

Reply:
I'm having trouble rebasing.  I sync'd my core:dev with yours, pulled my dev, 

```sh
git checkout dev
git pull
git checkout simplisafe-outdoor-camera-support 
git rebase dev
```

and I get a bunch of conflicts I did not expect.  I seem to be getting conflicts with previous commits on my simplisafe-outdoor-camera-support branch.

```text
First, rewinding head to replay your work on top of it...
Applying: Add support for Simplisafe outdoor cameras
Applying: change media urls to snake case
Applying: Exception checking.
Applying: update simplisafe-python version to 2024.01.0 for motion capture support
Using index info to reconstruct a base tree...
M	homeassistant/components/simplisafe/manifest.json
Falling back to patching base and 3-way merge...
No changes -- Patch already applied.
Applying: sync with proper EVENT_DISARMED
Using index info to reconstruct a base tree...
M	homeassistant/components/simplisafe/alarm_control_panel.py
Falling back to patching base and 3-way merge...
No changes -- Patch already applied.
Applying: add a base class for cameras that support motion, and an OutdoorCamera that uses this base class.
Applying: Add support for Simplisafe outdoor cameras
Using index info to reconstruct a base tree...
M	homeassistant/components/simplisafe/__init__.py
M	homeassistant/components/simplisafe/binary_sensor.py
M	homeassistant/components/simplisafe/services.yaml
Falling back to patching base and 3-way merge...
CONFLICT (add/add): Merge conflict in homeassistant/components/simplisafe/camera.py
Auto-merging homeassistant/components/simplisafe/camera.py
Auto-merging homeassistant/components/simplisafe/__init__.py
CONFLICT (content): Merge conflict in homeassistant/components/simplisafe/__init__.py
error: Failed to merge in the changes.
Patch failed at 0007 Add support for Simplisafe outdoor cameras
hint: Use 'git am --show-current-patch' to see the failed patch
Resolve all conflicts manually, mark them as resolved with
"git add/rm <conflicted_files>", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
```

camera.py for example, is not even in dev.  I am not sure what is happening.  Can you suggest anything?

Reply:
Run a `git status` and this stage and share your output.

Reply:
```sh
$ git status
On branch simplisafe-outdoor-camera-support
Your branch is up to date with 'origin/simplisafe-outdoor-camera-support'.

nothing to commit, working tree clean
```

```sh
$ git rebase dev
First, rewinding head to replay your work on top of it...
Applying: Add support for Simplisafe outdoor cameras
Applying: change media urls to snake case
Applying: Exception checking.
Applying: update simplisafe-python version to 2024.01.0 for motion capture support
Using index info to reconstruct a base tree...
M       homeassistant/components/simplisafe/manifest.json
Falling back to patching base and 3-way merge...
No changes -- Patch already applied.
Applying: sync with proper EVENT_DISARMED
Using index info to reconstruct a base tree...
M       homeassistant/components/simplisafe/alarm_control_panel.py
Falling back to patching base and 3-way merge...
No changes -- Patch already applied.
Applying: add a base class for cameras that support motion, and an OutdoorCamera that uses this base class.
Applying: Add support for Simplisafe outdoor cameras
Using index info to reconstruct a base tree...
M       homeassistant/components/simplisafe/__init__.py
M       homeassistant/components/simplisafe/binary_sensor.py
M       homeassistant/components/simplisafe/services.yaml
Falling back to patching base and 3-way merge...
CONFLICT (add/add): Merge conflict in homeassistant/components/simplisafe/camera.py
Auto-merging homeassistant/components/simplisafe/camera.py
Auto-merging homeassistant/components/simplisafe/__init__.py
CONFLICT (content): Merge conflict in homeassistant/components/simplisafe/__init__.py
error: Failed to merge in the changes.
Patch failed at 0007 Add support for Simplisafe outdoor cameras
hint: Use 'git am --show-current-patch' to see the failed patch
Resolve all conflicts manually, mark them as resolved with
"git add/rm <conflicted_files>", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".

$ git status
rebase in progress; onto 2b1d1340b7
You are currently rebasing branch 'simplisafe-outdoor-camera-support' on '2b1d1340b7'.
  (fix conflicts and then run "git rebase --continue")
  (use "git rebase --skip" to skip this patch)
  (use "git rebase --abort" to check out the original branch)

Unmerged paths:
  (use "git restore --staged <file>..." to unstage)
  (use "git add <file>..." to mark resolution)
        both modified:   homeassistant/components/simplisafe/__init__.py
        both added:      homeassistant/components/simplisafe/camera.py

no changes added to commit (use "git add" and/or "git commit -a")
```

Reply:
You need to see what changes Git struggles to reconcile in those two files, update accordingly, and then continue the rebase.

Reply:
```sh
$ git status
rebase in progress; onto 2b1d1340b7
You are currently rebasing branch 'simplisafe-outdoor-camera-support' on '2b1d1340b7'.
  (all conflicts fixed: run "git rebase --continue")

nothing to commit, working tree clean

$ git rebase --continue
Applying: Add support for Simplisafe outdoor cameras
No changes - did you forget to use 'git add'?
If there is nothing left to stage, chances are that something else
already introduced the same changes; you might want to skip this patch.
Resolve all conflicts manually, mark them as resolved with
"git add/rm <conflicted_files>", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
```

What state am I in now?

Reply:
When that occurs, that means that the current commit matches your working state—in that case, I would just `git rebase --skip`?

Reply:
Did you end up making a separate PR with the version bump and this websocket event name change?

Reply:
Yes